e2e/wdio/platform/tests/table.e2e-spec.ts

```diff
@@ -0,0 +1,669 @@
+
+        it('should check searching table item', () => {
+            findElementInTable(tableP13FilterExample, tableCellArr5);
+        });
+
+        it('should check filtering with include and exclude', () => {
+            scrollIntoView(tableP13FilterExample);
+            click(tableP13FilterExample + button, 1);
+            setValue(dialogInput, astroTestText);
+            click(expandedButton, 1);
+            click(popoverDropdownButton, 2);
+            click(filterByColorItem, 1);
+            setValue(dialogInput, testText7, 1);
+            click(footerButtonOk);
+
+            const rowLength = getElementArrayLength(tableP13FilterExample + tableRow);
+            expect(rowLength).toEqual(1);
+            expect(getText(tableP13FilterExample + tableRow + tableCell)).toBe(testText4);
+            expect(getText(tableP13FilterExample + tableRow + tableCell, 1)).toBe(testText5);
+
+        });
+
+        it('should check searching and placeholder in dialog', () => {
+            checkPlaceholder(tableP13FilterExample, 2);
+            checkSearchingInDialog();
+        });
+
+        it('should check sorting of columns', () => {
+            checkSortingColumns(tableP13FilterExample, 2);
+        });
+    });
+
+    describe('Check Grouping by multiple columns', function() {
+
+        it('should check searching table item', () => {
+            findElementInTable(tableP13GroupExample, tableCellArr4);
+        });
+
+        it('should check searching and placeholder in dialog', () => {
+            checkPlaceholder(tableP13GroupExample, 2);
+            checkSearchingInDialog();
+        });
+
+        it('should check sorting of columns', () => {
+            checkSortingColumns(tableP13GroupExample, 2);
+        });
+    });
+
+    describe('Check  Navigatable rows', function() {
+
+        it('should check clickable elements', () => {
+            scrollIntoView(tableNavigatableRowIndicatorExample);
+            click(tableNavigatableRowIndicatorExample + button);
+            expect(isElementClickable(tableNavigatableRowIndicatorExample + button, 1))
+                .toBe(true, 'button isnt clickable');
+
+            const rowLength = getElementArrayLength(tableNavigatableRowIndicatorExample + tableRow);
+            for (let i = 0; i < rowLength; i++) {
+                expect(isElementClickable(tableNavigatableRowIndicatorExample + tableRow, i))
+                    .toBe(true, `table row with index ${i} not clickable`);
+            }
+        });
+    });
+
+    describe('Check placeholders', function() {
+
+        it('should check placeholders in all input fields', () => {
+            const inputLength = getElementArrayLength(inputFields);
+            for (let i = 0; i < inputLength; i++) {
+                expect(getElementPlaceholder(inputFields, i)).toBe(placeholderTestText);
+            }
+        });
+    });
+
+    describe('Check playground', function() {
+
```

Comment:
no checks here for 
- sort / filter / grouping
- custom width
- changing title
- hide element count
![2021-09-20_3](https://user-images.githubusercontent.com/47522152/133992765-b92bf80c-a280-4ef4-9ee5-f08ba00269ed.png)